services/dummy-test-service/mongodb.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+// This is a user defined method to close resources.
```

Comment:
This file feels like copied code... do we need a licence copy at the top too?

Reply:
it was copied just to see if i was connecting to the DB. the comments will be erased :)